src/bit-systems/media-loading.ts

```diff
@@ -243,6 +247,9 @@ export function mediaLoadingSystem(world: HubsWorld) {
   });
 
   mediaLoaderExitQuery(world).forEach(function (eid) {
+    const loadingObjEid = loadingObjs.get(eid)!;
+    removeEntity(world, loadingObjEid);
+    loadingObjs.delete(eid);
```

Comment:
loadingObj is not set if `MEDIA_LOADER_FLAGS.ANIMATE_LOAD` flag is off, is it?

Reply:
Yes, that's correct `removeEntity` doesn't do anything if the eid is undefined but I'm going to add a check for code clarity.